src/Abstractions/ExecutionContext/BaseExecutionContext.cs

```diff
@@ -140,10 +141,16 @@ public BaseExecutionContext(IHostEnvironment? hostEnvironment = null)
 		/// </summary>
 		protected static string GetBuildVersion()
 		{
+			string buildVersion = DefaultEmptyValue;
```

Comment:
Can a regression test be added for this?

Reply:
Tried, original implementation shows the test runner version "17.8.0.0" without the hash so unit testing wouldn't catch the breaking change, it only affects our services at runtime.